apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -615,136 +573,60 @@ def from_anndata(
-                                conversions.to_tiledb_supported_array_type(
-                                    key, anndata.obsp[key]
-                                ),
-                                ingestion_params=ingestion_params,
-                                platform_config=platform_config,
-                                context=context,
-                                axis_0_mapping=jidmaps.obs_axis,
-                                axis_1_mapping=jidmaps.obs_axis,
-                            ) as sarr:
-                                _maybe_set(
-                                    obsp,
-                                    key,
-                                    sarr,
-                                    use_relative_uri=use_relative_uri,
-                                )
-
-                if len(anndata.varp.keys()) > 0:  # do not create an empty collection
-                    _util.uri_joinpath(measurement_uri, "obsp")
-                    with _create_or_open_collection(
-                        Collection,
-                        _util.uri_joinpath(measurement.uri, "varp"),
-                        ingestion_params=ingestion_params,
-                        context=context,
-                    ) as varp:
-                        _maybe_set(
-                            measurement, "varp", varp, use_relative_uri=use_relative_uri
-                        )
-                        for key in anndata.varp.keys():
-                            with _create_from_matrix(
-                                SparseNDArray,
-                                _util.uri_joinpath(varp.uri, key),
-                                conversions.to_tiledb_supported_array_type(
-                                    key, anndata.varp[key]
-                                ),
-                                ingestion_params=ingestion_params,
-                                platform_config=platform_config,
-                                context=context,
-                                axis_0_mapping=jidmaps.var_axes[measurement_name],
-                                axis_1_mapping=jidmaps.var_axes[measurement_name],
-                            ) as sarr:
-                                _maybe_set(
-                                    varp,
-                                    key,
-                                    sarr,
-                                    use_relative_uri=use_relative_uri,
-                                )
+                def _ingest_obs_var_m_p(
+                    ad_key: Literal["obsm", "varm", "obsp", "varp"],
+                    axis_0_mapping: AxisIDMapping,
+                    axis_1_mapping: Optional[AxisIDMapping] = None,
+                ) -> None:
+                    ad_val = getattr(anndata, ad_key)
+                    if len(ad_val.keys()) > 0:  # do not create an empty collection
+                        val_uri = _util.uri_joinpath(measurement_uri, ad_key)
```

Comment:
double-check: `measurement_uri` vs. `measurement.uri` here

Reply:
@ryan-williams for example
https://github.com/single-cell-data/TileDB-SOMA/blob/1.8.0/apis/python/src/tiledbsoma/io/ingest.py#L510-L517